tox.ini

```diff
@@ -35,7 +35,7 @@ deps =
     {[base]testdeps}
     coverage
 commands =
-    coverage run --source {[base]name} {envbindir}/nosetests
+    coverage run --source={[base]name} -m pytest tests
```

Comment:
isn't it simpler to use `pytest-cov`?

Reply:
There were reports that `pytest-cov` does not work properly on our Gitlab. So this way is for consistency and safety.

Reply:
Ah? Do you have a link to such report please? Because I am using it so I am wondering if I missed something?